VagrantFile

```diff
@@ -0,0 +1,67 @@
+#to UNIX EOL
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# specifies the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+# This file is built from the great work of Cristina Muñoz over at
+# https://github.com/PermanentOrg/devenv
+
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search or even make your own using
+  # the steps outlined at https://www.vagrantup.com/vagrant-cloud/boxes/create.
```

Comment:
Oh, hey, thanks for including this bit about creating one's own boxes.  I realize that the comments in this file aren't supposed to be an introduction to Vagrant, but I think this is worth including.